pkg/olm/operatorgroup.go

```diff
@@ -130,9 +173,16 @@ func (builder *OperatorGroupBuilder) Update() (*OperatorGroupBuilder, error) {
 	glog.V(100).Infof("Updating OperatorGroup %s in namespace %s",
 		builder.Definition.Name, builder.Definition.Namespace)
 
-	var err error
-	builder.Object, err = builder.apiClient.OperatorGroups(builder.Definition.Namespace).Update(
-		context.TODO(), builder.Definition, metav1.UpdateOptions{})
+	if !builder.Exists() {
+		return nil, fmt.Errorf("operatorGroup named %s in namespace %s does not exist",
```

Comment:
I think it's better to keep it generic like other pkgs:
```suggestion
		return nil, fmt.Errorf("cannot update non-existent operatorgroup",
```